### ISSUE TITLE
Set up CI / CD using GitHub Actions

### ISSUE THREAD

**Issue by fatchat** [DalgoT4D/dalgo-admin#10]

**Comment by GreatRSingh** [comment]
@fatchat Can i work on this issue.

**Comment by fatchat** [comment]
@GreatRSingh yes of course, but the project is nowhere near needing it right now

you can follow the project's progress and come back to this at the right time

**Comment by GreatRSingh** [comment]
> @GreatRSingh yes of course, but the project is nowhere near needing it right now
> 
> you can follow the project's progress and come back to this at the right time

ok ping me whenever you need.
